Program.cs

```diff
@@ -721,103 +722,117 @@ public static void Props()
+        //The idea is at base level it is simply attack and defence with a higher roll needed to hit if you are attacking someone defending. You can also pick up the knife or the crowbar which increase your damage
+        public static void Attack(int compHP)
         {
-            string answer;
-            int playeratt, playerHP, computeratt, computerHP, computer;
-            playeratt = 5;
-            playerHP = 25;
-            computerHP = 25;
-
+            bool done = false;
+            int i = 0;
+            string weapon;
+            string temp;
+            int playerDMG = 0;
+            int compDMG = 4;
+            int computerAction;
+            int toHit;
             Random rand = new Random();
-            computer = rand.Next(5);
-
-            switch (computer)
+            do //The loop for checking if there is a weapon in players inventory. Both weapons do the same damage so the first one found is used
             {
-                case 0:
-                    Console.WriteLine("computer attack");
-                    break;
-                case 1:
-                    Console.WriteLine("computer attack");
-                    break;
-                case 2:
-                    Console.WriteLine("computer attack");
-                    break;
-                case 3:
-                    Console.WriteLine("computer attack");
-                    break;
-                case 4:
-                    Console.WriteLine("computer defense");
-                    break;
-            }
-
-
-            while ((computerHP > 0) && (playerHP > 0))
+                if (inventory[i] == "crowbar" || inventory[i] == "knife")
+                {
+                    weapon = "a " + inventory[i];
+                    Console.WriteLine($"You have a {weapon} that can help you in the battle");
+                    done = true;
+                    playerDMG = 6;
+                }
+                i++;
+                if (i == inventory.Length)
+                { 
+                    done = true;
+                    playerDMG = 4;
+                }
+            } while (!done);
+            done = false;
+            do
             {
-
-                computeratt = rand.Next(1, 6);
-                Console.WriteLine($"Now, enemy hp is {computerHP:D2}");
-                Console.WriteLine("you want to attack or defend: ");
-                answer = Console.ReadLine();
-                computer = rand.Next(5);
-                switch (answer)
+                computerAction = rand.Next(0,2); // 0 is attack, 1 is defend
+                Console.WriteLine("Would you like to attack or defend?");
+                temp = Console.ReadLine().ToLower();
+                Console.Clear();
+                switch (temp)
                 {
                     case "attack":
-                        if ((computer == 0) || (computer == 1) || (computer == 2) || (computer == 3))
+                        if (computerAction == 0) //Both attack
```

Comment:
it would be cool to state what item is used to attack.

Reply:
Feature added.